lib/mssql.js

```diff
@@ -163,8 +134,22 @@ export async function check(req, res, pool) {
   return {ok: true};
 }
 
+export function addToPools(config, pools, {signal}) {
+  const pool = new mssql.ConnectionPool(config);
+  const close = pool.close.bind(pool);
+
+  pool.close = (...args) => {
```

Comment:
Why are we monkeypatching the library object here? Where else is connection.close being called?